src/test/java/com/philips/research/spdxbuilder/persistence/ort/OrtReaderTest.java

```diff
@@ -13,21 +13,43 @@
 import java.nio.file.Path;
 import java.util.List;
 
+import static junit.framework.TestCase.assertTrue;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 
 class OrtReaderTest {
-    private static final File ORT_SAMPLE = Path.of("src", "test", "resources", "ort_sample.yml").toFile();
 
-    private final BillOfMaterials bom = new BillOfMaterials();
-
-    @Test
-    void loadsOrtSample() {
-        new OrtReader(ORT_SAMPLE)
-                .defineProjectPackage("NPM::mime-types:2.1.18", URI.create("pkg:npm/mime-types@2.1.18"))
-                .excludeScopes("NPM::mime-types:2.1.18", List.of("test*"))
-                .read(bom);
-
-        assertThat(bom.getPackages()).hasSize(1 + 2);
-    }
+  private static final File ORT_SAMPLE1 = getPathOfFile("ort_sample1.yml");
+  private static final File ORT_SAMPLE2 = getPathOfFile("ort_sample2.yml");
+  private final BillOfMaterials bom = new BillOfMaterials();
+
+  static File getPathOfFile(String fileName) {
+	return Path.of("src", "test", "resources", fileName).toFile();
+  }
+
+  void createBOM(File file) {
+	OrtReader ortSample = new OrtReader(file);
+	ortSample.defineProjectPackage("NPM::mime-types:2.1.18", URI.create("pkg:npm/mime-types@2.1.18"))
+	  .excludeScopes("NPM::mime-types:2.1.18", List.of("test*"))
+	  .read(bom);
+  }
+
+  @Test
+  void loadsOrtSample() {
+	createBOM(ORT_SAMPLE1);
+	assertThat(bom.getPackages()).hasSize(1 + 2);
+  }
+
+  @Test()
+  void abortConversionOnIssues() {
+	Exception exception = assertThrows(OrtReaderException.class, () ->
+	  createBOM(ORT_SAMPLE2)
+	);
+
+	String expectedMessage = "The analyzed ORT file has issues, unable to generate a valid SPDX file";
+	String actualMessage = exception.getMessage();
+
+	assertTrue(actualMessage.contains(expectedMessage));
```

Comment:
All unit tests have been written as "fluent assertions" using AssertJ (see https://assertj.github.io/doc).

Please follow this style for consistency with the rest of the code.

For exceptions this would become something like:
```
assertThatThrownBy(() -> createBOM(ORT_SAMPLE2))
    .isInstanceOf(OrtException.class)
    .hasMessageContaining("file has issues");
```